egret/common/lazy_ptdf_utils.py

```diff
@@ -484,6 +510,22 @@ def _iter_over_int_viol_set(int_viol_set, mb, PTDF, abs_ptdf_tol, rel_ptdf_tol):
             mb.pfi[i_n] = expr
         yield i, i_n
 
+def _generate_branch_thermal_bounds(mb, bn, thermal_limit, neg_slacks, pos_slacks):
+    neg_slack, pos_slack = None, None
+    if neg_slacks and bn in mb.pf_slack_neg:
+        neg_slack = mb.pf_slack_neg[bn]
+    if pos_slacks and bn in mb.pf_slack_pos:
+        pos_slack = mb.pf_slack_pos[bn]
+    return libbranch.generate_thermal_bounds(mb.pf[bn], -thermal_limit, thermal_limit, neg_slack, pos_slack)
+
+def _generate_interface_bounds(mb, i_n, minimum_limit, maximum_limit, neg_slacks, pos_slacks):
+    neg_slack, pos_slack = None, None
```

Comment:
Why are `get_slack` and `pos_slack` hard coded?

Reply:
Similar to above.